MessageApp/ConversationListViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ConversationListViewController.swift
+//  MessageApp
+//
+//  Created by Macbook on 25.09.2022.
+//
+
+import UIKit
+
+final class ConversationListViewController: UIViewController {
+    
+    private lazy var tableView = UITableView(frame: .zero)
```

Comment:
```suggestion
    private let tableView = UITableView()
```

Тебе тут не нужен `lazy var`, т.к. во время инициализации ты не используешь никакие еще не существующие свойства